raiden/tests/integration/long_running/test_integration_events.py

```diff
@@ -339,15 +340,17 @@ def test_query_events(
         },
     )
 
-    events = get_token_network_registry_events(
-        chain=app0.raiden.chain,
-        token_network_registry_address=app0.raiden.default_registry.address,
-        contract_manager=contract_manager,
-        events=ALL_EVENTS,
-        from_block=999999998,
-        to_block=999999999,
-    )
-    assert not events
+    if blockchain_type == 'geth':
+        # TODO why do we query events in nonexisting blocks here?
```

Comment:
The test wants to confirm that querying for non-existing blocks returns an empty list.